allennlp/interpret/saliency_interpreters/smooth_gradient.py

```diff
@@ -0,0 +1,81 @@
+# pylint: disable=protected-access
+import math
+from typing import Dict, Any
+import torch
+import numpy
+from allennlp.common.util import JsonDict, sanitize
+from allennlp.interpret import SaliencyInterpreter
```

Comment:
the mypy error you're getting related to this line is most likely caused by a circular reference. try importing from the base module instead

```
from allennlp.interpret.blah.blah import SaliencyInterpreter
```